app/components/windows/go-live/GoLiveError.tsx

```diff
@@ -83,8 +83,6 @@ export default class GoLiveError extends TsxComponent<{}> {
         return this.renderRestreamError(error);
       case 'YOUTUBE_STREAMING_DISABLED':
         return this.renderYoutubeStreamingDisabled(error);
-      case 'YOUTUBE_PUBLISH_FAILED':
-        return this.renderYoutubePublishError(error);
```

Comment:
not sure why this is removed?

Reply:
This accidentaly came from my original big PR. YT should automatically publish the broadcast now, this error is not happening anymore